src/features/users/usersSlice.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-param-reassign */
+import { PayloadAction, createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+import { User } from '../../types/User';
+import { getUsers } from '../../api/users';
+
+export interface UsersState {
+  users: User[];
+  expanded: boolean,
+  userError: boolean,
```

Comment:
```suggestion
  error: boolean,
```

I think you can use `error` name and if need change it on destructing level:
```
const { error: userError } = useAppSelector(...);
```

Also you can think about `hasError` to highlight boolean type